core/store/orm/orm.go

```diff
@@ -335,6 +335,7 @@ func (orm *ORM) DeleteExternalInitiator(accessKey string) error {
 
 // FindExternalInitiator finds an external initiator given an authentication request
 func (orm *ORM) FindExternalInitiator(eia *models.ExternalInitiatorAuthentication) (*models.ExternalInitiator, error) {
+	// Why does this not mirror FindBridge?
```

Comment:
Not sure what this statement is asking, it looks mostly the same to me except using an accessKey instead of a bridge name.

Reply:
Well, its also using errors.Wrap(..), maybe it should be made consistent with the others and not wrap the error?  Or maybe the others should wrap theirs, with more error information.